Core/scripts/dirac-install.py

```diff
@@ -1267,6 +1284,8 @@ def installExternals( releaseConfig ):
     #HACK: try to find a more elegant solution for the lcg bundles location
     if not downloadAndExtractTarball( tarsURL + "/../lcgBundles", "DIRAC-lcg", verString, False, cache = True ):
       logERROR( "Check that there is a release for your platform: DIRAC-lcg-%s" % verString )
+      logERROR( "If you do not need LCG utils, use the option --no-lcg-bundle to skip LCG Bundle installation." )
```

Comment:
I would not make any economy on this error message. So, I suggest that this is a multiline message, something like:

The requested LCG software version %s for the local operating system is not found.
Please, check the availability of the LCG software bindings for you platform.

If you want to skip the installation of the LCG software, redo the installation 
with adding --no-lcg-bundle command line option.